extensions-jvm/pom.xml

```diff
@@ -51,7 +51,7 @@
         <module>aws2-translate</module>
         <module>cassandraql</module>
         <module>couchbase</module>
-        <module>debezium-mongodb</module>
+        <!--module>debezium-mongodb</module-->
```

Comment:
Would you please file a followup issue?

Reply:
https://github.com/apache/camel-quarkus/issues/1064